test/submission/submission_test.dart

```diff
@@ -46,6 +47,7 @@ Future<void> main() async {
     expect(submission.isSelf, isFalse);
     expect(submission.isVideo, isFalse);
     expect(submission.likes, isTrue);
+    expect(submission.vote, VoteState.upvoted);

```

Comment:
Just to confirm, you've run the tests correct? I _think_ this should be fine but I'd rather make sure :)